.bazelci/presubmit.yml

```diff
@@ -37,6 +37,14 @@ tasks:
       - "@go_sdk//..."
     test_targets:
       - "//..."
+  ubuntu2004_examples:
```

Comment:
I'm gonna add an issue to have the macOS build work for this, but I have zero ideas on how to configure it, and the buildkite site was not helpful.

Reply:
https://github.com/bazelbuild/rules_go/issues/3318 is filed